pom.xml

```diff
@@ -602,6 +602,19 @@
       <properties>
         <hadoop.version>1.0.4</hadoop.version>
       </properties>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-compiler-plugin</artifactId>
+            <configuration>
+              <excludes>
+                <exclude>**/AlluxioFileSystem.java</exclude>
+              </excludes>
+            </configuration>
+          </plugin>
+        </plugins>
+      </build>
```

Comment:
Is there another way we could specify this rule so that we only including `AlluxioFileSystem` if hadoop version is 2.0 or greater? Although this works with the profile, I think it would cause problems if people tried to compile with `-Dhadoop.version=1.21`